server/container/guice/mailbox-plugin-deleted-messages-vault-guice/src/main/java/org/apache/james/modules/vault/DeletedMessageVaultModule.java

```diff
@@ -46,6 +43,8 @@
 
     @Override
     protected void configure() {
+        install(new DeletedMessageVaultRetentionModule());
```

Comment:
why this split?

Reply:
There are two modules for binding DeletedMessageVault: one for binding MailRepository based vault, one for binding BlobStore based vault. and that both two modules need to load retention configuration -> this split.

Reply:
ok